bin/license-checker-ci

```diff
@@ -88,11 +97,11 @@ function checkLicense(licenseString, packageString) {
 		licenseString = licenseString.substring(0, licenseString.length - 1);
 	} */
 	return (
-		(acceptedSpdxExpr && spdxIds.includes(licenseString) && satisfies(licenseString, acceptedSpdxExpr))
-		|| acceptedNonSpdx.indexOf(licenseString) >= 0
+		acceptedNonSpdx.indexOf(licenseString) >= 0
 		|| packageString.startsWith(projectPackage.name + '@') // accept your own package
 		|| _.some(config.acceptedScopes, (scope) => packageString.startsWith(`@${scope}/`))
 		|| licenseString.startsWith(specialExemptionPrefix)
+		|| (acceptedSpdxExpr && satisfiesNoThrow(licenseString, acceptedSpdxExpr))
```

Comment:
Check all the non-standard stuff before invoking `satisfies` as it expects valid SPDX IDs.